### PR TITLE
Making it chrome compatible

### PR DESCRIPTION
**Pull request by rajatvijay**

### CODE REVIEW

File: src/devtools/devtools.js [hunk 1]
```diff
@@ -12,11 +20,14 @@ function handleHidden() {
   console.log("panel is being hidden");
 }
 
+// In chrome this function
+// doesn't return a promise
```
Comment (captainbrosset) [inline src/devtools/devtools.js]: Does the Firefox version accept a callback like Chrome?
In any case, we don't currently do anything useful here, so I suggest removing this whole part (removing the handleShown/handleHidden functions, and removing the callback altogether).

**Comment by rajatvijay** [comment]
> Do you mind taking care of my comment below please? 
Sure this is done.

**Comment by captainbrosset** [comment]
Thanks a lot! Merged.

**Comment by rajatvijay** [comment]
@captainbrosset 

Do you think we should update the readme for Browser support too?

**Comment by captainbrosset** [comment]
Yes, good point, I'll do this. I'll mention that although the code is chrome-compatible, the extension hasn't been pushed to the chrome web store.

**Comment by captainbrosset** [comment]
Done in e30d67fc3b197fa3d130863f53ebe08362c1f50b

**Comment by rajatvijay** [comment]
And let's push this to chrome web store too?


On Feb 22, 2018 15:19, "Patrick Brosset" <notifications@github.com> wrote:

[image: Boxbe] <https://www.boxbe.com/overview> This message is eligible
for Automatic Cleanup! (notifications@github.com) Add cleanup rule
<https://www.boxbe.com/popup?url=https%3A%2F%2Fwww.boxbe.com%2Fcleanup%3Fkey%3D%252BvhLXIWDjbvV3E8Szx0RrL155xjWZoncNvbmCtBLl5U%253D%26token%3DiQaFRGxRVb%252ByigPitIYA5XfKfrieIC6OGvKvISgph71h20WQINa0dRnd54JL6ouR%252F8Ni2M6Z7vEwPxebk7H%252B9hGJOHqMMNN9MclVv43EDKx3P%252BOH0lgxfp458P%252Fd5SBKVRcdgeYx1NUUOsCBh%252FpCgQ%253D%253D&tc_serial=36920502375&tc_rand=612349230&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001>
| More info
<http://blog.boxbe.com/general/boxbe-automatic-cleanup?tc_serial=36920502375&tc_rand=612349230&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001>

Yes, good point, I'll do this. I'll mention that although the code is
chrome-compatible, the extension hasn't been pushed to the chrome web store.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/captainbrosset/devtools-highlighter/pull/22#issuecomment-367593283>,
or mute the thread

**Comment by captainbrosset** [comment]
I tested it a little bit in Chrome and found several problems still, so I'll file issues for these and then we can push it to the chrome web store.

**Pull request merged**